docs/shared_parsers_catalog/getconf_page_size.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.getconf_pagesize
```

Comment:
The file name should be renamed to `getconf_pagesize.rst` in which the basename should be the same as the parser.